### DISCUSSION TITLE
Welcome to openpolicedata Discussions!

### DISCUSSION THREAD

**Discussion by sowdm** [openpolicedata/openpolicedata#60] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by trevisscout** [comment]


**Reply by potto216** [reply]
Hi, a great way to learn about this project is contributing to one of the open issues. Issues tagged with "good first issue" is a good place to start.
